doc/source/ray-air/benchmarks.rst

```diff
@@ -113,26 +113,26 @@ using a Pytorch pre-trained ResNet model.
 We test out the performance across different cluster sizes and data sizes.
 
 - `GPU image batch prediction script`_
-- `GPU training small cluster configuration`_
-- `GPU training large cluster configuration`_
+- `GPU prediction small cluster configuration`_
+- `GPU prediction large cluster configuration`_
 
 .. list-table::
 
     * - **Cluster Setup**
       - **Data Size**
       - **Performance**
       - **Command**
-    * - 1 g3.8xlarge node
+    * - 1 g4dn.8xlarge node
       - 1 GB (1623 images)
-      - 72.59 s (22.3 images/sec)
+      - 46.12 s (35.19 images/sec)
       - `python gpu_batch_prediction.py --data-size-gb=1`
-    * - 1 g3.8xlarge node
+    * - 1 g4dn.8xlarge node
       - 20 GB (32460 images)
-      - 1213.48 s (26.76 images/sec)
+      - 285.2 s (113.81 images/sec)
```

Comment:
g4dn provides decent speed up over g3 but this number shows we've come a long way in perf ;)